internal/server/models/destination.go

```diff
@@ -13,6 +13,9 @@ type Destination struct {
 	UniqueID   string `gorm:"uniqueIndex:idx_destinations_unique_id,where:deleted_at is NULL"`
 	LastSeenAt time.Time
 
+	Version    string
+	APIVersion string
```

Comment:
Should we include this field if there's a clear mapping of Version: APIVersion in our releases?

Reply:
I was on the fence about including this. It seems distinct enough to warrant including but can definitely be omitted for now